### DISCUSSION TITLE
Fresh deployed Docker instance - no login / registration possible?

### DISCUSSION THREAD

**Discussion by Innovathor69** [OpnForm/OpnForm#350] [Q&A]
Hey there,

i just deployed on railway.com a new docker of opnform (no errors). When i try to access via browser, i cant login or make a registration. Just nothing happens after i click register or login.

Are there any global variables have to set?  Is there a default user? A friend of me just had the same issue at a docker on debian server ...

Thanks for some help.

**Comment by danielsalinasgongora** [comment]
Please we really love this option to keep away unwanted users on self-hosted deploy.

**Reply by JhumanJ** [reply]
Hey guys! The behavior you describe is going to become the new default very soon. We just need to build the invite-only feature.

**Comment by ShonkaiDJ** [comment]
Are we havin fun here?

**Comment by LeeBuckland** [comment]
Not an answer per se, but hopefully something to help advance the learning here. I've been having some issues with my self-hosted server recently, so I thought I would build a new server to see if I can just get something up and running. Interestingly enough, I'm having the same issue of not being able to do the initial login. The only thing I can put this down to is that both my original server and the newly built server have the latest Linux updates installed. While I can't say for sure this is the issue, it is the only change that has occurred recently that would appear to be the likely candidate.

**Comment by JhumanJ** [comment]
Closing this as invite-only is the default in OpnForm self-hosted!

**Reply by LeeBuckland** [reply]
Does this mean there is a default username and password, given you can't create it on a fresh install?

**Reply by JhumanJ** [reply]
Absolutely: https://docs.opnform.com/deployment/docker#initial-login

**Reply by LeeBuckland** [reply]
Thank you - that was the missing piece I was unable to find originally.
